### ISSUE TITLE
Implement HPC execution

### ISSUE THREAD

**Issue by rvhonorato** [haddocking/haddock3#68]
Implement HPC execution

**Comment by joaomcteixeira** [comment]
Can we consider `HPC` fully implemented after #192, #193, #263, and #291 ?

**Comment by rvhonorato** [comment]
Does it have Slurm/Torque and GRID?

**Comment by rvhonorato** [comment]
> Can we consider `HPC` fully implemented after #192, #193, #263, and #291 ?

For clarification and prosperity
#192 is the implementation of the lib
#193 is a bugfix
#263 is also a bugfix
#291 is related to example/test

So the only implementation was done in #192, from there until now I don't know what changed in relation the parameters, but I trust your judgment everything is working fine.

Doing the GRID submission is worth its own issue and probably still far from now.
